app/components/works/edit_link_component.rb

```diff
@@ -11,6 +11,11 @@ def initialize(work:, anchor:, label:)
       @label = label
     end
 
+    sig { returns(T::Boolean) }
+    def render?
+      work.can_update_metadata? && !work.pending_approval?
```

Comment:
will this prevent a reviewer from making changes?

Reply:
oh, yes ... i'll ditch the `!work.pending_approval?` check